services-custom/dynamodb-enhanced/src/test/java/software/amazon/awssdk/enhanced/dynamodb/converters/attribute/InstantAsStringAttributeConvertersTest.java

```diff
@@ -79,12 +79,6 @@ public void InstantAsStringAttributeConverterNotAcceptLocalDateTimeTest() {
                                                                        .toAttributeValue()));
     }
 
-    @Test
-    public void InstantAsStringAttributeConverterNotAcceptOffsetTimeTest() {
-        assertFails(() -> transformTo(CONVERTER, EnhancedAttributeValue.fromString("1988-05-21T00:12:00+01:00")
-                                                                       .toAttributeValue()));
-    }
-
```

Comment:
Note to reviewer: JDK 17 now supports parsing offset times. There shouldn't be a problem supporting this now, so I removed the assertion that we do not support it.